### ISSUE TITLE
make a global task list too.

### ISSUE THREAD

**Issue by aasutossh** [nocksock/do.nvim#7]
Currently `.do_tasks` is created (maybe at project root).
changing projects (`:Telescope projects`), still persists the Do s.
Changing projects should empty the Do s?

**Comment by nocksock** [comment]
You're right, this should be the behaviour. I got some time on friday and will look into that.

**Comment by nocksock** [comment]
Thought about this again recently, and deliberately deciding against this now. Once you started on a dofile, it should be the same dofile for that session.
